testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/microprofile/restclient/RestClientProviderPriorityTest.java

```diff
@@ -31,6 +33,7 @@
 
 @RunWith(Arquillian.class)
 @RunAsClient
+@Category(NotForBootableJar.class) // no RX layer so far
```

Comment:
> no RX layer so far

Where is RX classes used here?

Reply:
You're right. There's no use of rx classes, but I added the annotation because I noticed a rx related error. Maybe I was too quick, I am pushing changes that remove it and we'll see what happens, although this should not be considered at all when running `ts.bootable` profile since I noticed that you always used `-Dts.disable.microprofile...` right?

Reply:
> this should not be considered at all when running ts.bootable profile since I noticed that you always used -Dts.disable.microprofile... right?

There is no `ts.disable.microprofile`. There is `disable.microprofile.tests`, but it affects tests with `MicroProfileDependent`. But this RestClientProviderPriorityTest class is not annotated by `MicroProfileDependent`, so it's not related.

Reply:
> There is no ts.disable.microprofile. There is disable.microprofile.tests

Yeah, I was typing quickly and didn't go look for the name but I see you got it.

> But this RestClientProviderPriorityTest class is not annotated by MicroProfileDependent, so it's not related.

Good to know. Annotations were removed, let's see what CI checks say.

Reply:
@marekkopecky this test is passing now, can we resolve this conversation?